### DISCUSSION TITLE
Multiple Desks in the Ward

### DISCUSSION THREAD

**Discussion by ageekhere** [CorsixTH/CorsixTH#2424] [Research Room]
Something that is unclear for me is what do multiple Desks in the Ward do? Does having multiple nurses in a Ward speed up patient turn around?

My understanding would be that by having more than 1 nurse in a Ward it would lead to
1. Faster patient diagnosis
2. Less down time when a nurse goes to the staff room, by having 2 or 3 desks we can be quite sure that a nurse is always in the Ward

**Issues with multiple Desks and nurses for the Ward**
If you do have say 2 Desks with 2 nurses and one goes to the staff or another room they will not return to the Ward if a nurse is already in there because they are not needed so they wait in the hall.

**What should happen** 
Nurses should go to Wards that have empty desks in them if all rooms that need nurses have a nurse allocated to them rather then waiting in the halls.

Thoughts?

**Comment by tobylane** [comment]
Each nurse adds to the speed of healing. [Ward:updateHealingFactor](https://github.com/CorsixTH/CorsixTH/blob/87858c2d5a4060a6039ca4110fb24403d17d4123/CorsixTH/Lua/rooms/ward.lua#L127) for the exact details.

Do nurses use second desks like that in original Theme Hospital?

**Reply by ageekhere** [reply]
Ok so having more than one desk is a strategy, then the issue where not all of the nurses will return to the Ward after staff room is a bug.
